airflow/utils/cli.py

```diff
@@ -366,3 +368,12 @@ def _wrapper(*args, **kwargs):
                     logging.disable(logging.NOTSET)
 
     return cast(T, _wrapper)
+
+
+def validate_dag_bundle_arg(bundle_names: list[str]) -> None:
+    """Make sure only known bundles are passed as arguments."""
+    known_bundles = {b.name for b in DagBundlesManager().get_all_dag_bundles()}
+
+    non_existant_bundles: set[str] = set(bundle_names) - known_bundles
```

Comment:
since i made another suggestion already...
```suggestion
    unknown_bundles: set[str] = set(bundle_names) - known_bundles
```
they might exist.... somewhere??? :)